x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/common/types.ts

```diff
@@ -30,6 +30,10 @@ export interface Field {
   type: KBN_FIELD_TYPES;
 }
 
+export interface RuntimeMapping {
+  type: string;
+  script: any;
```

Comment:
Are there different ways to define scripts or could it be `{ source: string; }`?

Reply:
I removed this interface altogether and use RuntimeField from `data` plugin instead here. I think that's probably better considering the API can keep changing. https://github.com/elastic/kibana/pull/90363/commits/3cfed188b4eb9b64340ae9f852ca6867dc147f05